applicationinsights.ts

```diff
@@ -64,6 +68,15 @@ export function setup(instrumentationKey?: string) {
         Logging.info("The default client is already setup");
     }
 
+    if (!liveMetricsClient) {
```

Comment:
Shouldn't this only get set up if enabled?

Reply:
The way it is now, it is only constructed. Enablement would happen at `AppInsights.start()`. I did it this way for consistency with `defaultClient`. This is probably not necessary and can be changed if you think it should only be constructed if enabled.

Reply:
I'm just thinking a default telemetry client is always necessary to be available, but the live metrics stuff is just eating memory and compute without a purpose in the default state if this is made in `setup`

Reply:
Refactored how client is constructed. Currently constructed on first call of `setSendLiveMetrics`. Subsequent calls enable/disable it. Calling `start()` will enable the client if it exists and should be enabled